lib/rubocop/cop/style/commented_keyword.rb

```diff
@@ -57,7 +57,7 @@ class CommentedKeyword < Base
 
         REGEXP = /(?<keyword>\S+).*#/.freeze
 
-        SUBCLASS_DEFINITION = /\A\s*class\s+\w+\s*<\s*\w+/.freeze
+        SUBCLASS_DEFINITION = /\A\s*class\s+[\w:]+\s*<\s*[\w:]+/.freeze
```

Comment:
I think using `::` makes it clearer as an identifier. What do you think?

```suggestion
        SUBCLASS_DEFINITION = /\A\s*class\s+(\w|::)+\s*<\s*(\w|::)+/.freeze
```

Reply:
@koic 
Thanks for the suggestion!!
I adopted it because it's clearer than my implement.